bank/src/main/java/org/example/account/Account.java

```diff
@@ -0,0 +1,115 @@
+package org.example.account;
+
+
+import java.math.BigDecimal;
+
+
+public class Account {
+
+    public static int howmanyaccout = 0;
+    public String id;
+    public String pw;
+    protected String category;
+    protected String accountNum;
+    protected BigDecimal balance = new BigDecimal(0);
+    protected String owner;
+    protected boolean isVaild;
+
+
+
+    public Account(String id,String pw, String owner, String accountNum, String category, BigDecimal balance) {
+        this.id =id;
+        this.pw= pw;
+        this.owner = owner;
+        this.accountNum =accountNum;
+        this.category=category;
+        this.balance=balance;
+        howmanyaccout++;
+    }
+
+    public boolean SuccessLogin(String id, String pw) {
```

Comment:
```suggestion
    public boolean successLogin(String id, String pw) {
```
카멜케이스를 지켜주세요.
https://tecoble.techcourse.co.kr/post/2020-04-26-Method-Naming/